scipy/optimize/_optimize.py

```diff
@@ -3089,7 +3163,9 @@ def myfunc(alpha):
         return ((fval, p, xi) if fval is not None else (func(p), p, xi))
     elif lower_bound is None and upper_bound is None:
         # non-bounded minimization
-        alpha_min, fret, _, _ = brent(myfunc, full_output=1, tol=tol)
+        res = _recover_from_bracket_error(_minimize_scalar_brent,
```

Comment:
Not super familiar with this part of the code base so wanted to check that the switch from `brent` to `_minimize_scalar_brent` was intentional?

Reply:
Yes. I don't remember the original reason to do this, but `brent` just calls `_minimize_scalar_brent`, so we might as well just call `_minimize_scalar_brent`. Also, use of `_minimize_scalar_brent` is a little more consistent with use of `_minimize_scalar_bounded` below.